.circleci/config.yml

```diff
@@ -186,6 +198,10 @@ workflows:
           filters:
             tags: # only runs on tags if you specify this filter
               only: /.*/
+      - 'test-go-windows':
+          filters:
+            tags:
+              only: /.*/
       - 'package':
```

Comment:
shouldn't 'package' and 'release' also require test-go-windows?

Reply:
Good catch, added `test-go-windows` as requirements for package and release